deps/v8/build/toolchain.gypi

```diff
@@ -1104,7 +1104,7 @@
           'VCCLCompilerTool': {
             'Optimization': '0',
             'conditions': [
-              ['component=="shared_library"', {
+              ['component=="shared_library" or node_shared=="true"', {
```

Comment:
We will submit to V8, do we just need to change node_shared to force_dynamic_crt as part of doing that ?